src/styles/learn.scss

```diff
@@ -2,7 +2,6 @@
   /* Needed since overflow: none will disable position: sticky on children */
   clip-path: polygon(0 0, 100% 0, 100% 100%, 0 100%);
   background-color: var(--color-fill-side-nav);
-  height: calc(100vh - var(--nav-height));
```

Comment:
These changes are seemingly breaking the `/learn` page on mobile (https://staging.nodejs.dev/1447/learn)

![image](https://user-images.githubusercontent.com/54291836/122652869-cceac100-d15e-11eb-8fff-1dd2cd502c16.png)

Can you keep the styling changes limited to `about.scss` only

Reply:
#1448 is taking care of the style issues

Reply:
The cross button is not visible in the deployed version (https://nodejs.dev/learn), which restricts the user from closing the sidenav in mobile view.

Reply:
#1448 handled that as well.

Reply:
Great, thanks.

Reply:
Here the pending thing, is you need to revert these lines to their original state.

Reply:
Reverted back the above-mentioned changes and changes for extender class as both of these are being handled in #1448